CODEOWNERS

```diff
@@ -355,6 +355,7 @@
 /subsys/bluetooth/                        @joerchan @jhedberg @Vudentz
 /subsys/bluetooth/controller/             @carlescufi @cvinayak @thoh-ot
 /subsys/bluetooth/mesh/                   @jhedberg @trond-snekvik @joerchan @Vudentz
+/subsys/canbus/                           @alexanderwachter
```

Comment:
@henrikbrixandersen Should you place yourself as an owner to files at subsys/canbus/canopen?

Reply:
@jukkar @henrikbrixandersen asked me to use my github handle for the canbus subsystem.
I think its a good idea not to split the CAN maintainership. We can add him as a collaborator in the new maintainer file then.